pyvista/core/dataset.py

```diff
@@ -2523,3 +2522,49 @@ def cell_type(self, ind: int) -> int:
 
         """
         return self.GetCellType(ind)
+
+    def point_is_inside_cell(self, ind: int, point: Vector) -> bool:
+        """Return whether a point is inside a cell.
+
+        Parameters
+        ----------
+        ind : int
+            Cell ID.
+
+        point : Sequence[float] or numpy.ndarray
+           Length 3 sequence defining a point in space.
+
+        Returns
+        -------
+        bool
+            Whether point is inside cell.
+
+        Examples
+        --------
+        >>> from pyvista import examples
+        >>> mesh = examples.load_hexbeam()
+        >>> mesh.cell_bounds(0)
+        (0.0, 0.5, 0.0, 0.5, 0.0, 0.5)
+        >>> mesh.point_is_inside_cell(0, [0.2, 0.2, 0.2])
+        True
+
+        """
+        if ind < 0 or ind >= self.n_cells:
```

Comment:
Nitpick:
```suggestion
        if not 0 <= ind < self.n_cells:
```
at least I always find it easier to reason about a single interval, despite the negation.